ui/src/actions/actions-ui.js

```diff
@@ -48,9 +48,12 @@ export const updateDisabledAppVersion = (id, isDisabled) => {
   };
 };
 
-export const toggleAppDetailedIsDirty = () => {
+export const setAppDetailedIsDirty = (isDirty) => {
```

Comment:
Suggested function name: `setAppDetailedDirtyState`. 

`setAppDetailedIsDirty` sounds like a one way action - setting `isDirty` to true.